popup.js

```diff
@@ -8,7 +8,6 @@ const channelNameInput = document.querySelector('.js-channel-name-input')
 const randomButton = document.querySelector('#random')
 const thumbnailVideoDropzone = document.querySelector('.py-label-video-thumbnail')
 const thumbnailInput = document.querySelector('.js-thumbnail-input')
-const errorMessageSpan = document.querySelector('#extErrorMessage')
```

Comment:
Error handling is not used anymore in this file, the `.startsWith('https://www.youtube.com/')` check was the only possible error here and it's now handled in the main extension popup